src/Pelagos/Bundle/AppBundle/Resources/views/layout-head.html.twig

```diff
@@ -0,0 +1,41 @@
+{% block head %}
+<head>
```

Comment:
I think the head tag should be outside the block, because if you want to overwrite the head block, the tags should still be there.